packages/ui-react/src/components/LoginForm/LoginForm.tsx

```diff
@@ -58,6 +59,11 @@ const LoginForm: React.FC<Props> = ({ onSubmit, onChange, socialLoginURLs, value
       <SocialButtonsList socialLoginURLs={socialLoginURLs} />
       <h2 className={styles.title}>{t('login.sign_in')}</h2>
       {errors.form ? <FormFeedback variant="error">{errors.form}</FormFeedback> : null}
+      {success ? (
+        <FormFeedback visible={false} variant="success">
+          {t('login.sign_in_success')}
+        </FormFeedback>
+      ) : null}
```

Comment:
I think we go a bit overboard by announcing a successful login. The most important thing is that in every form, that when a mistake happens, the "error" is state announced. Because this is (most likely) unexpected behaviour. 

This should be the main priority for every form. 

Other cases where a "succes" announcement is handy, when users `could` expect a follow up message. Such as:
- When you submitted a contact form (eg "we will be in touch with you shortly")
- When subscribing to a newsletters (eg "you have been subscribed")
- etc

A login succesfull or "the form is correctly saved" in other cases doesn't add a lot of value. This last example could also not hurt, but because we're limited on time let's prioritze!

Reply:
I understand your concern, using the implementation proposed by @ChristiaanScheermeijer i've managed to implement the announcer throughout our forms in a time efficient manner.

Reply:
Really nice!